src/cuga/backend/server/tool_guard_generation.py

```
@@ -0,0 +1,83 @@
+                },
+            )
+            policy_code = await generation_agent.policies.generate_tool_guard_code(
+                policy_id=policy_id,
+                target_tool=tool_name,
+            )
+            await generation_agent.policies.update_tool_guard(
+                policy_id=policy_id,
+                tool_guards={tool_name: {"policy_code": policy_code}},
+            )
+            results.append({"tool": tool_name, "status": "ok"})
+        except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)